### ISSUE TITLE
[Components and pattern standards] Design component or pattern has not been tested with Veterans and/or VA.gov users following Platform research guidelines. (04.13.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#61756]
#### VFS team name
Appointments

#### VFS product name
Appointment List

#### VFS feature name
Print List

#### Point of Contact/Reviewers
Allison Christman - @allison0034 - Design

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Design component or pattern has not been tested with Veterans and/or VA.gov users following Platform research guidelines.

### Issue Details
Since we originally started the experimental design process with this button, it has shifted and we have tested it in the context of using it with a card. We are concerned it may not have the affordance of a button outside of a card. 

### VA.gov Experience Standard
[Category Number 04, Issue Number 13](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
Contributing to the design system, Platform Research Overview

### Platform Recommendation
Since this is a new design and the context has changed, we would like to get some testing with Veterans before we release it on VA.gov

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by outerpress** [comment]
Notes from 07/19 collab cycle office hours (copying from [61757](https://github.com/department-of-veterans-affairs/va.gov-team/issues/61757#issuecomment-1642506562)) 

- We (VAOS team) will release the button as designed and start gathering click analytics.
@humancompanion-usds will work on iterations to the button/link situation - I'll send a Sketch version of our detail page redesign WIP so he can explore the more complex use-cases
- Once an iteration is ready, we'll modify the print button and compare analytics.
- When we're ready with all MHV on VA.gov features slated for September, we'll run UAT which will include the print button.
- We'll report back to design council with our findings for that, and decide if further iteration is needed.)

We'll keep this ticket open to track analytics and iterations that come up. FYI @ldelacosta 

- We (VAOS team) will release the button as designed and start gathering click analytics.
@humancompanion-usds will work on iterations to the button/link situation - I'll send a Sketch version of our detail page redesign WIP so he can explore the more complex use-cases
Once an iteration is ready, we'll modify the print button and compare analytics.
When we're ready with all MHV on VA.gov features slated for September, we'll run UAT which will include the print button.
We'll report back to design council with our findings for that, and decide if further iteration is needed.

**Comment by outerpress** [comment]
FYI @allison0034 @humancompanion-usds [here's how the tertiary button is performing so far](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/research/2023-08-print-button-analytics/research-report.md#print-button-analytics-and-feedback---august-2023). 

TL;DR: 

> **Immediately on release, users started clicking the print button.** Desktop users clicked it the most often, and most users printed the upcoming appointments page. We believe this indicates that the print button is findable/usable. 
> [Go to the detailed finding](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/research/2023-08-print-button-analytics/research-report.md#google-analytics-usage-data)

> **August was the first month since November 2022 where no users left comments related to printing.** We believe that, because Medallia feedback trends negative, receiving no feedback is an indicator that the print button may be meeting the intended need. 
> [Go to the detailed finding](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/research/2023-08-print-button-analytics/research-report.md#medallia-feedback)

**Comment by outerpress** [comment]
@allison0034 @humancompanion-usds we've tested this with Veterans and saw no issues. [Here's the link to the finding.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/research/2023-11-mhv-on-va-gov-phase-2b-vaos-uat/research-findings.md#all-participants-were-able-to-easily-use-the-print-feature). 

I'm happy to add more detail, let me know if you have any questions. If not, I think this can be closed (cc @JRRoof  @shiragoodman )

<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNEsrQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--cc473bc1bf24a494b1bb51e6ab06bb239a03b4f1/Screenshot%202024-01-11%20at%203.24.02%E2%80%AFPM.png" alt="Screenshot 2024-01-11 at 3.24.02 PM.png" />

**Comment by shiragoodman** [comment]
@outerpress looks like Allison has no concerns per her thumbs up emoji response. You should be good to close!

**Comment by outerpress** [comment]
Thank you @shiragoodman ! Fyi @ldelacosta
